data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/teleport/teleport.cfg

```diff
@@ -0,0 +1,46 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [teleport]
+##
+# Actions:
+# 
+##
+# Expected end state:
+# 
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "teleport" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY_NO_VALUE teleport ()}
+                [/abilities]
+            [/effect]
+        [/modify_unit]
+
+        [move_unit]
+            id=alice
+            to_x=3
+            to_y=3
+        [/move_unit]
+    [/event]
+
+    [event]
+        name=moveto
+
+        [store_unit]
+            [filter]
+            [/filter]
+            variable=temp
+        [/store_unit]
+
+        #{ASSERT {VARIABLE_CONDITIONAL temp[1].status.uncovered boolean_equals no}}
+        #{SUCCEED}
```

Comment:
I guess this is why the PR is still in draft, adding a comment to remember it.

Reply:
Yeah, I figured I might as well open it in case there were any comments on the skirmisher or hides tests, since I'm not sure how in depth I want to get with the [tunnel] testing.

Reply:
The teleports tests have been added.